bayesian/api/api_v2.py

```diff
@@ -303,6 +305,24 @@ def stack_analyses():
         raise HTTPError(500, e.args[0])
 
 
+@api_v2.route('/stack-report/<crda_key>/<stack_id>', methods=['GET'])
```

Comment:
Is it okay to print and share the uuid?

Reply:
I feel uuid is a sensitive value., shouldn't be shared as part of this.

Reply:
We can mask it, base64 encoded or whatever

Reply:
I think sharing UUID is not an issue. It is required when you are using CI so why should it be different here?

Reply:
In CI, user/admin takes care copying it to CI specific secrets.

However when we display it over cli, user can share it with external party which is not desirable to know the UUID.

Reply:
We decided to remove UUID from URL, and handle it in API.

Reply:
Update: RDS query to fetch crda_key corresponding to particular stack _id will be made in Redirect URL function only.